clang/lib/AST/MicrosoftMangle.cpp

```diff
@@ -3552,7 +3552,22 @@ void MicrosoftCXXNameMangler::mangleType(const DependentSizedExtVectorType *T,
 
 void MicrosoftCXXNameMangler::mangleType(const ConstantMatrixType *T,
                                          Qualifiers quals, SourceRange Range) {
-  Error(Range.getBegin(), "matrix type") << Range;
+  QualType EltTy = T->getElementType();
+  const BuiltinType *ET = EltTy->getAs<BuiltinType>();
```

Comment:
You should also backport ccdd55c518277d749eff878ffcb5ca3de55c2a60 or manually delete this unused variable to keep the release branch CI green.

Reply:
Is there a backport request for this?

Reply:
I've added this commit to the PR now.